restler/utils/logger.py

```diff
@@ -671,9 +773,18 @@ def add_hash(replay_filename):
                 bug_bucket = bug_buckets[bucket_class][seq_hash]
                 bucket_hash = f"{seq_hash}_{bucket_class}"
                 name_header = bucket_class
+                checkerName = bucket_class[0:0+ bucket_class.find("_"+bucket_class.split("_")[len(bucket_class.split("_"))-1])]
                 if bucket_hash not in Bugs_Logged:
                     try:
                         filename = log_new_bug()
+                        filenameJson = log_new_bug_as_json()
+                        requestBug = Bug()
+                        requestBug.filepath = filenameJson
+                        requestBug.reproducible = bug_bucket.reproducible
+                        requestBug.checkerName = checkerName
+                        requestBug.errorCode =bucket_class.split("_")[len(bucket_class.split("_"))-1] 
```

Comment:
>bucket_class.split("_")[len(bucket_class.split("_"))-1]  [](http://example.com/codeflow?start=46&length=56)

How will the error code be used?  Do we want the actual error code (`201`), or the one from the bucket class (`20x` in some cases)?

Reply:
The error code is similar to the content in bug_buckets.txt "InvalidValueChecker_500 - Bug was reproduced - InvalidValueChecker_500_1.txt" this line is converted to individual properties  checkername , filepath, reproducible, error_code